tools/uswds-component-library-figma.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: USWDS Component Library in Figma
+nav: Tools
+permalink: /tools/uswds-component-library-figma
+sidenav: tools-side-nav
+---
+
+# {{ page.title }}
+
+The [USWDS component library](https://www.figma.com/community/file/836611771720754351) is available and free to use through the Figma Community. Created by the [Truss.works team](https://truss.works/), it's a great place to start incorporating components into your designs.
+
+## Getting started
+
+### Duplicating the USWDS component library
+
+To view and duplicate files from the Figma Community, login or create a [Figma account](https://www.figma.com/).
+
+1. Navigate to the community tab in the left-hand side navigation
+1. Search for "USWDS" in the search bar
+1. Open the USWDS file by Truss
+1. Duplicate the file
+
+![GIF demonstrating how to navigate through Figma's community page to duplicate the U.S. Web Design System library created by Truss.]({{ site.url }}/{{site.baseurl}}/assets/img/uswds-component-library-getting-started.gif)
+
+### Publishing the library
+
+You've successfully duplicated the library! Now you can publish this file and apply it across all your project files.
+
+![GIF demonstrating how to navigate through Figma's community page to duplicate the U.S. Web Design System library created by Truss.]({{ site.url }}/{{site.baseurl}}/assets/img/uswds-component-library-publishing-file.gif)
+
+## Start crafting designs 
+
+Not sure where to start or need inspiration? 
+
+We've gone ahead and created some page templates using the USWDS components library. Check them out [here](#)!
+<!--need to link to USWDS page templates later-->
+
+<!-- ## Pro tips:
+- You can make changes to any of the components on your duplicated library file without affecting Truss's original file.  -->
+
+## Things to keep in mind:
+- Whichever version of the library you duplicated from Truss will **not** receive any new updates automatically.
+- The only way to get the newest version would be to duplicate the updated library from the Figma Community page.
+- If you have files or projects already linked to an older version, tedious relinking may occur if you're wanting to move to an updated library version.
+
+### Our suggestions:
+- **Keep your USWDS component library in it's own file.** If your team or projects have multiple files referencing from this library, it's best to keep it separate.
+- **Stick to a single version.** It's best to stick to a single version of the USWDS components library to avoid continuous relinking of components.
+
+## Figma-related resources
```

Comment:
Let's try just "Resources" here, as that's something that could end up being consistent across the all the pages.